lib/install/operation.go

```diff
@@ -217,7 +217,7 @@ func (i *Installer) getClusterStatus() (*status.Status, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	status, err := status.FromCluster(i.ctx, clusterOperator, *cluster, "")
+	status, err := status.FromCluster(i.ctx, clusterOperator, *cluster, "", "")
```

Comment:
I agree with @r0mant that we should at least move these to a configuration struct to minimize impact.